evap/evaluation/templates/comment_visibility_buttons.html

```diff
@@ -0,0 +1,19 @@
+{% load i18n %}
+{% load evaluation_templatetags %}
+
+<div class="btn-group" data-toggle="buttons">
+    {% for choice in form.comment_visibility %}
+        {% if choice.choice_value == "ALL" %}
+            {% trans "Will see all comments about the course and about all contributors." as tooltip %}
+        {% elif choice.choice_value == "COURSE" %}
+            {% trans "Will see own comments and comments about the course." as tooltip %}
+        {% elif choice.choice_value == "OWN" %}
+            {% trans "Default value. Will only see own comments." as tooltip %}
+        {% endif %}
+        {% if form.responsibility.value == "RESPONSIBLE" %}
+            {% include_choice_button form.comment_visibility choice False tooltip %}
+        {% else %}
+            {% include_choice_button form.comment_visibility choice True tooltip %}
```

Comment:
i think variable assignment is only possible with `with`and something like `{% with disabled = form.responsibility.value == "RESPONSIBLE" %}` does not work. if you have any other idea, please let me know. and keep in mind that we would actually need `enabled`, and as you know django template doesn't like negations.